### DISCUSSION TITLE
My application for GitHub Student got rejected for no reason, and now I can't reapply again

### DISCUSSION THREAD

**Discussion by ahmedafsa** [community/community#143377] [GitHub Education]
### Hi everyone,

I submitted a request to join the GitHub Student Pack on October 29. It was initially approved by the bot on the application page, but I received an email yesterday stating my application was rejected without a specific reason and advising me to reapply.

However, whenever I try to access the [GitHub Education Page](https://education.github.com/discount_requests/application), I keep getting redirected to this [pricing page](https://github.com/pricing).

Could you please clarify the issue and guide me on how to reapply? I am pretty sure that I submitted everything correctly.

Thanks in advance

**Comment by InWamos** [comment]
Bot can't give you the approval. This is how approval looks like:
![image](https://github.com/user-attachments/assets/bc08d604-1d32-47b8-beea-12210d427937)
[Go here](https://education.github.com/discount_requests/application?type=student) and you should see the column labeled "You've already submitted n requests". Beneath this sign you will find the reason for rejection

**Reply by ahmedafsa** [reply]
Yes I know, already got the green label on this page under "You've already submitted n requests",
but now whenever I try to open it I'm redirected to this pricing page

![image](https://github.com/user-attachments/assets/be590250-15f4-4220-b50f-77c32f54a86e)

**Comment by Gvolexe** [comment]
Hi there,

That does seem a bit odd. I recommend trying to log in via an incognito tab or private browsing mode to access the GitHub Education page. Sometimes, cached information can cause unexpected redirects.

If that doesn’t resolve the issue, it would be best to contact GitHub Education support directly. You can reach them at:
**support@github.com**

Explain the situation, including the dates and the email you received, and they should be able to assist you further.

Hope this helps, and good luck with your reapplication!
